nexus-repository-conan/src/main/java/org/sonatype/repository/conan/internal/metadata/ConanHashVerifier.java

```diff
@@ -55,14 +55,14 @@ public HashCode lookupHashFromAsset(final StorageTx tx, final Bucket bucket, fin
     return null;
   }
 
-  private AttributesMap getAttributes(final StorageTx tx, final Bucket bucket, final String assetPath) {
+  private AttributesMap getConanmanifestHashes(final StorageTx tx, final Bucket bucket, final String assetPath) {
```

Comment:
`getConanmanifestHashes` -> `getConanManifestHashes`

Reply:
I agree but there is conanmanifest.txt file in Conan world. So I will keep as it.